core/src/rpc.rs

```diff
@@ -2873,7 +2873,7 @@ pub mod tests {
         let largest_accounts: Vec<RpcAccountBalance> =
             serde_json::from_value(json["result"]["value"].clone())
                 .expect("actual response deserialization");
-        assert_eq!(largest_accounts.len(), 19);
+        assert_eq!(largest_accounts.len(), 20);
```

Comment:
Minor changes to account for the fact that the spl-token 2 native mint account now blinks into existence when in OperatingMode::Development...